.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -10,18 +10,15 @@
 
 # Definition of Done
   Refer to [Asteroids DoD document](https://salesforce.quip.com/iq2mAy4i62oM) to see any additional details for the items below
-- [ ] Place holder data is incorporated (Refer to [Case Management Sample Data document](https://quip.com/cbFcAPJF8t0z))
-- [ ] Base axe-core JEST test included for any new LWC (No critical violations)
+- [ ] Any net new LWC work has Sa11y tests & 50% or above lines JEST test coverage  
```

Comment:
Heads up, if not already aware: future revisions of the SFDO DoD are likely to start imposing Jest coverage requirements closer to 90%, but other teams are well behind y'all in adoption so there is considerable time to ramp up.